### PR TITLE
UI Improvements

### LINKED ISSUE #969: In Questionnaire form, Some Questions are bold in text and some are normal in text, Unique format for questionnaire is not there
**Describe the bug**
In Questionnaire form, Some Questions are bold in text and some are normal in text, Unique format for questionnaire is not there

**Component**
SDC gallery app

**To Reproduce**
1. Go to 'Data Capture Gallery'
2. Click on 'General Health Assessment Questionnaire'
3. Observe questionnaire text format

**Expected behavior**
Unique format for all questionnaire should be there

**Screenshots**
Screenshot_2021-12-14-12-43-58-845

**Smartphone (please complete the following information):**
 - Device: Nokia 3.4
 - Android version: 11

### PR DESCRIPTION
**Pull request by jingtang10**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #969 

**Description**

* Fixes the incorrect bold styling for auto complete widget
* Enforces better naming convention (uses snake_case) for views in layout files
* Remove unused file `QuestionnaireItemCheckBoxViewHolderFactory.kt`
* Remove test file `QuestionnaireItemCheckBoxViewHolderFactoryInstrumentedTest.kt`
* Remove an unnecessary layer of `LinearLayout` in `questionnaire_item_check_box_view.xml`

**Alternative(s) considered**

NA

**Type**

Bug fix + Code health

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md)
- [x] I have read [How to Contribute](https://github.com/google/android-fhir/blob/master/docs/contributing.md)
- [x] I have read the [Developer's guide](https://github.com/google/android-fhir/wiki/Developer's-Guide)
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate )
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally
- [x] I have built and run the reference app(s) to verify my change fixes the issue and/or does not break the reference app(s)

### CODE REVIEW

File: datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactory.kt [hunk 1]
```diff
@@ -49,13 +47,13 @@ internal object QuestionnaireItemDatePickerViewHolderFactory :
...
-        textInputLayout = itemView.findViewById(R.id.textInputLayout)
-        textInputEditText = itemView.findViewById(R.id.textInputEditText)
+        prefixTextView = itemView.findViewById(R.id.prefix_text_view)
+        textDateQuestion = itemView.findViewById(R.id.question_text_view)
+        textInputLayout = itemView.findViewById(R.id.text_input_layout)
+        textInputEditText = itemView.findViewById(R.id.text_input_edit_text)
         // Disable direct text input to only allow input from the date picker dialog
         textInputEditText.keyListener = null
-        textInputEditText.setOnFocusChangeListener { view: View, hasFocus: Boolean ->
+        textInputEditText.setOnFocusChangeListener { _: View, hasFocus: Boolean ->
```
Comment (aditya-07) [inline datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactory.kt:56]: nit:
```suggestion
        textInputEditText.setOnFocusChangeListener { _, hasFocus: Boolean ->
```

**Review by aditya-07 — COMMENTED** [review]
Apart from the nit, LGTM

**Review by aditya-07 — APPROVED** [review]

File: datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactory.kt (hunk 1 above)
Comment (jingtang10) [reply datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactory.kt:56]: will be handled in my next pr

Comment (jingtang10) [reply datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactory.kt:56]: sorry missed this

**Pull request merged**